core/daemon/src/main/resources/META-INF/opennms/applicationContext-daemon.xml

```diff
@@ -55,6 +55,7 @@
     init-method="start" destroy-method="stop">
     <property name="maxConnections" value="#{T(java.lang.System).getProperty('org.opennms.activemq.client.max-connections', '8')}" />
     <property name="idleTimeout" value="#{T(java.lang.System).getProperty('org.opennms.activemq.client.idle-timeout', '30000')}" />
+    <property name="reconnectOnException" value="#{T(java.lang.System).getProperty('org.opennms.activemq.client.reconnect-on-exception', 'true')}" />
```

Comment:
If we add a new option here, we should also add it in the Blueprint:
 https://github.com/OpenNMS/opennms/blob/ee96be8496af43550188622abcc0934c25d95bcf/features/activemq/component/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L41

This pool definition is only used by OpenNMS, and the other is used by Minion.